src/sdk/modules/validators/smartSessionValidator/decorators/enableSessions.ts

```diff
@@ -0,0 +1,224 @@
+import type {
+  Account,
+  ActionData,
+  PolicyData,
+  Session
+} from "@rhinestone/module-sdk"
+import type { Chain, Client, Hex, PublicClient, Transport } from "viem"
+import {
+  type GetSmartAccountParameter,
+  type SmartAccount,
+  sendUserOperation
+} from "viem/account-abstraction"
+import {
+  encodeAbiParameters,
+  encodeFunctionData,
+  getAction,
+  parseAccount
+} from "viem/utils"
+import { UniActionPolicyAbi } from "../../../../__contracts/abi"
+import { SmartSessionAbi } from "../../../../__contracts/abi/SmartSessionAbi"
+import addresses from "../../../../__contracts/addresses"
+import { AccountNotFoundError } from "../../../../account/utils/AccountNotFound"
+import type { CreateSessionDataParams } from "../../../utils/Types"
+import {
+  createActionConfig,
+  createActionData,
+  generateSalt,
+  getPermissionId,
+  toActionConfig,
+  toTimeRangePolicy
+} from "./Helper"
+import type {
+  EnableSessionsActionReturnParams,
+  EnableSessionsResponse
+} from "./Types"
+// Review: Execution type could either be used from our sdk or from module-sdk
+
+const UNIVERSAL_POLICY_ADDRESS = addresses.UniActionPolicy
+const SIMPLE_SESSION_VALIDATOR_ADDRESS = addresses.SimpleSessionValidator
+
+export type EnableSessionsParameters<
+  TSmartAccount extends SmartAccount | undefined
+> = GetSmartAccountParameter<TSmartAccount> & {
+  sessionRequestedInfo: CreateSessionDataParams[]
+  maxFeePerGas?: bigint
+  maxPriorityFeePerGas?: bigint
+  nonce?: bigint
+  signatureOverride?: Hex
+}
+
+export const getSmartSessionValidatorEnableSessionsAction = async ({
+  sessionRequestedInfo,
+  client,
+  account // TODO: review type
+}: {
+  sessionRequestedInfo: CreateSessionDataParams[]
+  client: PublicClient
+  account: Account // TODO: review type
+}): Promise<EnableSessionsActionReturnParams | Error> => {
+  console.log("account", account)
```

Comment:
log

Reply:
build was failing so I logged it, as it is unused.
similar utils in module sdk passes account : Account 
can remove it for now

Reply:
done